Paintroid/build.gradle

```diff
@@ -41,18 +41,19 @@ if (project.hasProperty('independent')) {
 }
 
 android {
-    compileSdkVersion 25
-    buildToolsVersion '25.0.3'
+    compileSdkVersion 26
+    buildToolsVersion '26.0.2'
```

Comment:
buildToolsVersion is ignored with android-gradle-plugin versions >= 3
this line can be removed

Reply:
Thanks, removed the `buildToolsVersion` line